test/e2e/framework/volume_util.go

```diff
@@ -196,6 +199,14 @@ func NewRBDServer(cs clientset.Interface, namespace string) (config VolumeTestCo
 		},
 	}
 	pod, ip = CreateStorageServer(cs, config)
+
+	// Ceph server container needs some time to start. Tests continue working if
+	// this sleep is removed, however kubelet logs (and kubectl describe
+	// <client pod>) would be cluttered with error messages about non-existing
+	// image.
+	Logf("sleeping a bit to give ceph server time to initialize")
```

Comment:
there's no health check we can define and check to make this actually accurate?

Reply:
None that I know about. The image we use expects the container to initialize a fresh pool and imports an 100MB volume during startup which takes some time.

In the long run we could ship already prepared image, but last time I checked it was too big for me (I don't remember exact numbers though).

Reply:
If the tests continue working regardless, then I guess it doesn't matter if this is too short.